src/modm/platform/can/windows_hosted/can.cpp.in

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright (c) 2011-2012, 2017, Fabian Greif
```

Comment:
Is this CAN implementation really specific to Windows? I suspect this is applicable for all hosted targets.

Reply:
no, renamed the folder to just /hosted. If we want linux to have this as well, we just change the hosted-linux .xml file to include can as a module, which we can do later once we get a ful linux build tested, but I don't want to add stuff without a test setup to make sure my stuff actually works.

Reply:
:+1: FWIW, once you want to test it, it's pretty straightforward to enable WSL on Windows and test it on your machine. I was doing all my initial modm work in WSL due to bugs in their template generation at the time.